os/include/tinyara/usb/usbhost.h

```diff
@@ -108,23 +121,21 @@
  *
  ************************************************************************************/
 
-#define CLASS_CREATE(reg, drvr, id) ((reg)->create(drvr, id))
+#define CLASS_CREATE(reg,hport,id) ((reg)->create(hport,id))
```

Comment:
Coding rules: there should be a space after comma .

Reply:
Done for all the occurrences